staging/src/k8s.io/sample-controller/main.go

```diff
@@ -38,6 +38,9 @@ var (
 )
 
 func main() {
+        //Init the klog flags
```

Comment:
I think this comment can be removed (doesn't add much) but lgtm otherwise.

Reply:
ok, I will  remove this comment.
Thx for your reply.